### ISSUE TITLE
OutpostStation Pipenet Mapping

### ISSUE THREAD

**Issue by chairbender** [unitystation/unitystation#2812]
## Description
For #2805 , depends on #2811 , #2810 .

Map the pipenet into outpost station. The scrubbers / airvents should be set up so they are actually attached to a pipenet.

TODO: This may also require implementing other atmos objects in order to ensure that stuff is being properly distributed through the station and people aren't gasping.

**Comment by corp-0** [comment]
closing in favor of #4874 and because pipes are done already with the newer system.
